### ISSUE TITLE
Ensure Xaml intellisense generates when opening legacy projects with CPS

### ISSUE THREAD

**Issue by davidwengier** [dotnet/project-system#5529]
For https://github.com/dotnet/project-system/issues/2488 we fixed the WPF targets but that only applies to SDK projects.

As we bring up support for opening legacy projects in CPS we need to ensure we fix the issue for those projects too, probably by making them use the WPF targets from the SDK. See https://github.com/dotnet/wpf/pull/1895#issuecomment-532344926

**Comment by maxinfet** [comment]
@davidwengier Is this ticket able to be closed given the state of the two linked tickets?

**Comment by davidwengier** [comment]
No, those tickets have solved the issue for SDK-style projects, but legacy projects opened with CPS (the Common Project System that SDK-style projects run on top of) will still exhibit the problem.

Note that at the moment you have to work really hard and know what you're doing in order for this scenario to occur, so its most likely not anything you need to worry about. In general using WPF with SDK-style or legacy projects will work fine.
